django/db/models/constants.py

```diff
@@ -1,6 +1,12 @@
 """
 Constants used across the ORM in general.
 """
+from enum import Enum
 
 # Separator used to split filter strings apart.
 LOOKUP_SEP = '__'
+
+
+class OnConflict(Enum):
+    Ignore = 'ignore'
+    Update = 'update'
```

Comment:
Please uppercase the enum member names:
```suggestion
    IGNORE = 'ignore'
    UPDATE = 'update'
```